### PR TITLE
fix: disable essential telemetry until onboarded

### PR DESCRIPTION
**Pull request by chrmod**

### CODE REVIEW

File: extension-manifest-v3/src/background/telemetry/index.js [hunk 1]
```diff
@@ -113,9 +114,16 @@ chrome.runtime.onMessage.addListener((msg) => {
     log('Telemetry recordUTMs() error', error);
   }
 
-  if (JUST_INSTALLED) {
-    telemetry.ping('install');
-  }
-  telemetry.ping('active');
+  observe('terms', async (terms) => {
+    telemetryEnabled = terms;
```
Comment (smalluban) [inline extension-manifest-v3/src/background/telemetry/index.js:118]: This function runs at least once every background run. You don't need a flag, just check terms and call pings.

**Review by smalluban — CHANGES_REQUESTED** [review]

File: extension-manifest-v3/src/background/telemetry/metrics.js [hunk 2]
```diff
@@ -390,7 +379,7 @@ class Metrics {
         this.storage[`${type}_${frequency}`] = timeNow;
         this.saveStorage(this.storage);
```
Comment (philipp-classen) [inline extension-manifest-v3/src/background/telemetry/metrics.js:380]: Here, we are missing an `await`. IMHO, better to send too few messages than too many.

File: extension-manifest-v3/src/background/telemetry/index.js (hunk 1 above)
Comment (chrmod) [reply extension-manifest-v3/src/background/telemetry/index.js:118]: telemetry is always there, as it is used to set the uninstall url

**Review by smalluban — APPROVED** [review]
Ok,we can leave it as it is until we change the uninstall URL setup. 

I strongly suggest to not initialize telemetry just for setting the uninstall URL, which before the onboarding should be clean, so don't need to go through the telemetry at all.

**Review by philipp-classen — APPROVED** [review]

**Comment by chrmod** [comment]
@smalluban I completely agree. In next iteration I will re-write telemetry code from the ground-up.

**Pull request merged**